styles/components/buttons.css

```diff
@@ -0,0 +1,82 @@
+.btn {
+  padding: 0.5rem 1rem;
+  margin: 0.5rem;
```

Comment:
use 'px' in place of rem for values less that 1rem as suggested by Vikram sir

Reply:
Yes 😀, I also thought of the same, with my button's font size increasing w/ respect to root font size, I also wanted my padding and margin also to increase and decrease so that it could fit the text properly.  I think the usage of rems for less than 1rem is optional, not mandatory 🤔.